src/App.tsx

```diff
@@ -1,10 +1,84 @@
-import logo from './logo.svg';
 import './App.css';
 import Draw from './Views/Draw/Draw';
-import React, { useState } from 'react';
+import Settings from './Views/Settings/Settings';
+import { useEffect, useState } from 'react';
+import { generateContrastColors } from '@adobe/leonardo-contrast-colors';
+
+
+const defaultColors = {
```

Comment:
I'm going to merge in these changes. I am going to create a new sketchCanvasData-like-object to hold all of these color values, and pass them into the canvas component.